BlueShift-iOS-SDK/BlueshiftLog.m

```diff
@@ -0,0 +1,96 @@
+//
+//  BlueshiftLog.m
+//  BlueShift-iOS-SDK
+//
+//  Created by Ketan Shikhare on 29/07/20.
+//  Copyright © 2020 Bullfinch Software. All rights reserved.
```

Comment:
this should be Blueshift instead of Bullfinch

Reply:
This gets created automatically on file creation, the organization is set as Bullfinch in SDK Xcode project. I have modified the project organization name and the files to reflect Blueshift name.